consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java

```diff
@@ -345,7 +349,39 @@ private int overwriteLightblue(List<JsonNode> documentsToOverwrite) {
     protected Map<String, JsonNode> getDestinationDocuments(Map<String, JsonNode> sourceDocuments) {
         Map<String, JsonNode> destinationDocuments = new LinkedHashMap<>();
         if(sourceDocuments == null || sourceDocuments.isEmpty()){
-            LOGGER.info("Unable to fetch any destination documents as there are no source documents");
+            //LOGGER.info("Unable to fetch any destination documents as there are no source documents");
```

Comment:
Good catch, sorry about that. This is not part of the intended changes. Reverted.